src/components/__stories__/cells/ColumnPinningCell.tsx

```diff
@@ -0,0 +1,63 @@
+import React from 'react';
+
+import type {HeaderContext} from '@tanstack/react-table';
+
+import type {Item} from '../types';
+
+export interface ColumnPinningHeaderCellProps {
```

Comment:
No need to export interfaces from storybook components. They are not used anywhere else.